test/ranefterm.jl

```diff
@@ -0,0 +1,56 @@
+using MixedModels, StatsModels
```

Comment:
Should the tests "RandomEffectsTerm" and  "Categorical Blocking Variable" be moved from `FactorReTerm.jl` to here?